src/screenFlags.py

```diff
@@ -92,6 +95,13 @@ def getArgParser():
 of, say, deleted files in git status that you would like to restore to a given
 revision. It enables you to select the deleted files even though they
 do not exist on the system anymore.''')
+        parser.add_argument('-ai',
+                            '--all-input',
+                            default=False,
+                            action="store_true",
+                            help='''You may force PathPicker to recognize all
+lines as acceptible input. Typically, PathPicker will scan the input for references
+to file paths. Passing this option will disable those scans and the program will assume that every input line is a match.''')
```

Comment:
could add an explanation that its helpful for things like git branches, mercurial bookmarks, maybe even directories -- who knows!